examples/gke-a3ultragpu-staging.yaml

```diff
@@ -0,0 +1,181 @@
+# Copyright 2024 "Google LLC"
```

Comment:
The blueprint will be moved to the GoB repo. Kept here for making the review easier.

Reply:
Will delete from here